### PR TITLE
#184581354 tracking order status

### PR DESCRIPTION
**Pull request by Habihirwe**
**What does this PR do?**
tracking order status 
**Description of Task to be completed?**
buyer should be able to track  order status and admin should be able to update status then after real time update should be sent to the front end.
**How should this be manually tested?**
- npm i && npm run migrate:all
- to  get the status of the buyer here token must be provided 
- http://localhost:5000/order/status/
- admin to update the status , his or her token should be provided and order id that you want to update 
- http://localhost:5000/order/(orderId)
- testing realtime update
- http://localhost:5000/public/sse-test.html
**Any background context you want to provide?**
n/a
What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/story/show/184581354

**Screenshots (if appropriate)**
Screen Shot 2023-04-06 at 17 22 25
Screen Shot 2023-04-06 at 17 23 54
Screen Shot 2023-04-06 at 17 24 07

Questions:

### CODE REVIEW

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by Martha-Iradukunda — APPROVED** [review]
That's great.

**Review by blackd44 — APPROVED** [review]
LG

File: public/sse-test.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        //         return (
+        //           prev + `<tr><td>${curr.id}</td><td>${curr.status}</td></tr>`
+        //         );
+        //       }, '');
+        //       document.body.innerHTML = `
+        //   <table>
+        //     <thead><th>Order Id</th><th>Status</th></thead>
+        //     <tbody>
+        //       <tbody>
+        //         ${orderData}
```
Comment (UmuhireJessie) [inline public/sse-test.html]: Remove this!

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
check the comment

File: public/sse-test.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          transform: rotate(360deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="spinner"></div>
+    <script>
+      function fetchData() {
+        const sse = new EventSource(`http://localhost:5000/order/sse`);
```
Comment (edwinnambaje) [inline public/sse-test.html]: if also possible you can remove the localhost as it can't work in hosted version

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**